### ISSUE TITLE
Can't use \sabbrev functions

### ISSUE THREAD

**Issue by darrSonik** [akinomyoga/ble.sh#70]
**ble version**: 0.4.0-devel2+4395484

**Bash version**: 5.0.18(1)-release, x86_64-pc-linux-gnu

I implemented `\branch` and `\commit` from the [recipe page](https://github.com/akinomyoga/ble.sh/wiki/Recipes) in your wiki. It worked fine when I first yanked those from your wiki. I didn't check it for a long time, but last night, I did `ble-update` and after that, `\branch` and `\commit` stopped working.

It spits:
```
-bash: ble/complete/candidates/filter:/test: No such file or directory
```
Screenshot:

![Error-BLE.sh](https://user-images.githubusercontent.com/34137568/100363525-117a0800-3027-11eb-802d-dfc107e080f5.png)
***
By the way, static abbreviations like `ble-sabbrev L='| less'` is working fine.

**Comment by akinomyoga** [comment]
Thank you for the report! It seems the bug has been introduced by the commit 7c6b67b. I fixed it in the latest devel branch dabc515. Could you check it by `ble-update`? Thank you again!

**Comment by darrSonik** [comment]
I did `ble-update` and it's working as great as it was.
Thanks!
